rbac/server/api/roles.py

```diff
@@ -181,6 +311,32 @@
 
 
 @ROLES_BP.delete("api/roles/<role_id>")
+@doc.summary("Delete a role by its next_id.")
+@doc.description("Delete a role by its next_id.")
```

Comment:
Should also mention this endpoint can only be used by NEXT admins and role owners